src/components/forms/Switch/Switch.stories.tsx

```diff
@@ -1,32 +1,145 @@
-import React from 'react';
 import { Meta, Story } from '@storybook/react/types-6-0';
+import React, { useState } from 'react';
 
+import { generateControl } from '../../../utils/tests/storybook';
+import { Stack } from '../../layout/Stack';
+import { Text } from '../../typography';
 import Switch from './Switch';
+import { SwitchSizes } from './Switch.enums';
 import { SwitchProps } from './Switch.types';
 
 export default {
   title: 'components/forms/Switch',
   component: Switch,
+  argTypes: {
+    maxWidth: { control: { type: 'number' } },
+    defaultChecked: { control: { type: 'boolean' } },
+    size: {
+      ...generateControl('select', SwitchSizes),
+    },
+  },
 } as Meta;
 
 export const playground: Story<SwitchProps> = (args) => <Switch {...args} />;
 playground.parameters = {
   chromatic: { disable: true },
 };
-playground.args = { label: 'Public', switchId: 'playground' };
+playground.args = {
+  label: 'Public',
+  switchId: 'playground',
+};
 playground.argTypes = {
   label: {
     description: 'Text in switch',
   },
   switchId: {
     description: 'ID to connect input and its label',
   },
+  maxWidth: {
+    description: 'Max width in px',
+  },
 };
 
-export const Default: Story = () => (
-  <Switch label="Private" switchId="default" />
+export const Default: Story = () => <Switch switchId="default" />;
+
+export const WithoutLabel: Story = () => (
+  <Stack gap="lg">
+    <Stack justify="flex-start">
+      <Text>Not Checked</Text>
+      <Switch switchId="withoutLabel1" />
+    </Stack>
+
+    <Stack justify="flex-start">
+      <Text>Not Checked and disabled</Text>
+      <Switch switchId="withoutLabel2" isDisabled />
+    </Stack>
+
+    <Stack justify="flex-start">
+      <Text>Checked</Text>
+      <Switch switchId="withoutLabel3" defaultChecked />
+    </Stack>
+
+    <Stack justify="flex-start">
+      <Text>Checked and disabled</Text>
+      <Switch switchId="withoutLabel4" defaultChecked isDisabled />
+    </Stack>
+  </Stack>
 );
 
-export const Checked: Story = () => (
-  <Switch label="Public" switchId="checked" defaultChecked />
+export const WithLabel: Story = () => (
+  <Stack gap="lg">
+    <Stack justify="flex-start">
+      <Switch label="False" switchId="withLabel1" />
+    </Stack>
+
+    <Stack justify="flex-start">
+      <Text>Not Checked and disabled</Text>
+      <Switch label="False" switchId="withLabel2" isDisabled />
+    </Stack>
+
+    <Stack justify="flex-start">
+      <Text>Checked</Text>
+      <Switch label="True" switchId="withLabel3" defaultChecked />
+    </Stack>
+
+    <Stack justify="flex-start">
+      <Text>Checked and disabled</Text>
+      <Switch label="True" switchId="withLabel4" defaultChecked isDisabled />
+    </Stack>
+  </Stack>
 );
+
+export const WithFixedWidthAndShortText: Story = () => (
+  <Stack gap="lg">
```

Comment:
You don't need to wrap the component in Stacks when there is only one child in it

Reply:
Yep. The copy paste feature 👎